sunbird_service_api_test/src/test/java/org/sunbird/integration/test/location/LocationTest.java

```diff
@@ -353,6 +376,40 @@ public void testDeleteDistrictLocation(String responseJson, String testName) {
     }
   }
 
+  @Test(
+      dataProvider = "stateBulkUploadDataProvider"
+  )
+  @CitrusParameters({"requestJson", "responseJson", "testName"})
+  @CitrusTest
+  /**
+   * Method to validate the functional test cases for the state type location bulk upload. It include scenarios -
+   * 1. Bulk upload file consists of valid data means all mandatory fields are available and we expecting success response.
```

Comment:
Change  Bulk upload file consists of valid data means all mandatory fields are available and we expecting success response  to  State upload with valid file. Expecting success response with processId.